.github/workflows/daily.yml

```diff
@@ -52,14 +64,27 @@ jobs:
         run: |
           poetry install
 
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
```

Comment:
Can we please pin this to a hash or at least a tag? I'd rather not give a 3rd party action my full Docker credentials.